### PR TITLE
Changed background definition (Fixes issue #757)

### LINKED ISSUE #757: [Bug] Description of `background` Policy Setting is Misleading
### Page link

https://kyverno.io/docs/writing-policies/policy-settings/#:~:text=background%3A%20controls%20if%20rules%20are%20applied%20to%20existing%20resources%20during%20a%20background%20scan.%20Defaults%20to%20%E2%80%9Ctrue%E2%80%9D

### Description

> `background`: controls if rules are applied to existing resources during a background scan. Defaults to “true”.

I think there are 2 levels of issues/solutions here either:
1) Semantically the word "rules" is too generic and should read "validation rules" and the word "applied" is just incorrect according to the below documentation. Doing some word replacement here might be the quick win.
2) The entire `background` description should be rewritten/updated to better match what seems to be the current source of truth (ie. the doc below).

https://kyverno.io/docs/writing-policies/background/

### Expected behavior

This is answered in the description.

### Slack discussion

https://kubernetes.slack.com/archives/CLGR9BJU9/p1612459243238900

### PR DESCRIPTION
**Pull request by yrs147**
## Related issue #
Closes #757

## Proposed Changes

Changed definition of background attribute for better understanding.

Before 
image

After
image

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md).
- [x] I have inspected the website preview for accuracy.
- [x] I have signed off my issue.

### CODE REVIEW

**Comment by yrs147** [comment]
@chipzoller

File: content/en/docs/Writing policies/policy-settings.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ A [policy](/docs/kyverno-policies/) contains one or more rules, and the followin
 
 * **validationFailureActionOverrides**: a ClusterPolicy attribute that specifies `validationFailureAction` Namespace-wise. It overrides `validationFailureAction` for the specified Namespaces.
 
-* **background**: controls if rules are applied to existing resources during a background scan. Defaults to "true".
+* **background**: controls scanning of existing resources to find potential violations and creating report of those violations during a background scan and is enabled by default.
```
Comment (chipzoller) [inline content/en/docs/Writing policies/policy-settings.md:15]: "controls scanning of existing resources to find potential violations and generating Policy Reports. See the documentation here (insert link). Default to "true""

**Review by chipzoller — CHANGES_REQUESTED** [review]

**Comment by chipzoller** [comment]
Please sign off on all commits.

File: content/en/docs/Writing policies/policy-settings.md (hunk 1 above)
Comment (yrs147) [reply content/en/docs/Writing policies/policy-settings.md:15]: 👍

**Comment by yrs147** [comment]
@chipzoller I think it's ready to merge

**Review by chipzoller — APPROVED** [review]

**Pull request merged**